src/components/criticalErrorMessage.jsx

```diff
@@ -15,4 +17,22 @@ const CriticalErrorMessage = () => (
   </p>
 );
 
-export default CriticalErrorMessage;
+export const DefaultErrorMessage = props => (
+  <ErrorPanel
+    className={props.style}
+    error="Something went wrong, please try again later."
+  />
+);
+
+export const MissingDataErrorMessage = props => (
+  <ErrorPanel
+    className={props.style}
+    error="This item has been missing data for at least 3 days."
```

Comment:
Instead of creating two components that are doing the same thing, I would store these two error messages in variables or as keys in an object and pass those in as props directly to `ErrorPanel`, or have some conditional logic that determines which message to show based on the error code.